_docs/getting-started/mac.md

```diff
@@ -29,41 +29,9 @@ pip install caffe2
 
 To compile Caffe2 to use your GPU, follow [these](#gpu-support) instructions first, then:
 
-* Follow [these](#anaconda-install-path) instructions if you have Anaconda.
-* Follow [these](#brew-and-pip-install-path) instructions if you do not have Anaconda.
-* (Advanced) Follow [these](#custom-anaconda-install) instructions if you have Anaconda, are used to working with Anaconda environments, are familiar with how Anaconda installs and finds dependencies, and need to change Caffe2 c++ source and recompile frequently.
+We recommend that you use [Anaconda](https://www.continuum.io/downloads). Activate Anaconda, create a new environment, install dependencies
```

Comment:
Nit: `Activate Anaconda, create a new environment, install dependencies` -> `Activate Anaconda, create a new environment, and install dependencies.`